SonarQube.Common/AnalysisProperties/Property.cs

```diff
@@ -36,8 +36,20 @@ public class Property
 
         #endregion
 
+        #region Public methods
+
+        /// <summary>
+        /// Returns whether the property contains any sensitive data
+        /// </summary>
+        public bool ContainsSensitiveData()
+        {
+            return ProcessRunnerArguments.ContainsSensitiveData(this.Id) || ProcessRunnerArguments.ContainsSensitiveData(this.Value);
+        }
+
+        #endregion
+
```

Comment:
This code existed in a couple of places. I've added it is a helper method on property for convenience.